packages/components/modules/profiles/web/profile-popover/AddProfileMenuItem/CreateProfileModal/index.tsx

```diff
@@ -55,6 +56,37 @@ const CreateProfileModal: FC<CreateProfileModalProps> = ({
     onClose?.()
   }
 
+  const parseErrorMessage = (
+    error: Error,
+  ): { field: 'name' | 'urlPath' | 'root'; message: string } => {
+    const errorMessage = error.message.toLowerCase()
+
+    if (errorMessage.includes('duplicate key') && errorMessage.includes('urlpath')) {
+      return {
+        field: 'urlPath',
+        message: 'This URL path is already taken. Please choose a different one.',
+      }
+    }
+    if (errorMessage.includes('duplicate key') && errorMessage.includes('name')) {
+      return {
+        field: 'name',
+        message: 'An organization with this name already exists. Please choose a different name.',
```

Comment:
@rf2tsl @anicioalexandre we should rethink this implementation, the backend should return the error messages well formatted for the frontend to show them correctly, attach to the correct field or whatever, we should think about building this "generic" way of communication, like the `setFormApiErrors(form, err)` function, it will make it easier to manage the text being shown and also internationalization.

Another issue is that we are trying to guess that backend means by checking the strings that it includes, it can be misleading, like for example `An organization with this name already exists.` this is not a constraint on the backend, you can create organizations with duplicated names.